src/routes/web.php

```diff
@@ -32,13 +34,31 @@
 });
 
 Route::middleware(['auth:sanctum', 'verified'])->get('/dashboard', function (Request $request) {
-    // Redirect to the dashboard for admins or homepage for other authenticated users; eventually we'll have a user dashboard
```

Comment:
This code all looks good. I would suggest moving this to a controller, though, so that the routes file doesn't get too big. You can run `php artisan make:controller StatsController` and the `make web-shell` prompt, move this code to a function in the controller named `getDashboard`, and then change this route file.

I would also suggest we make this open to the public, though I know the pre-existing route wasn't publicly accessible. Here's what the line would look like after moving this to the controller:

`Route::get('/dashboard', [App\Controllers\StatsController::class, 'getDashboard'])->name('dashboard');`